AdyenComponents/BLIK/BLIKComponent.swift

```diff
@@ -111,6 +117,12 @@ public final class BLIKComponent: PaymentComponent, PresentableComponent, Loadin
         return item
     }()
 
+    // MARK: - Private
+
+    private func sendTelemetryEvent() {
+        adyenContext.analyticsProvider.trackTelemetryEvent(flavor: telemetryFlavor)
```

Comment:
This same function is repeated many times. Can't we move it into the telemetry protocol and have a default extension? Or another protocol for this